src/parseRuntimeAndValidateSDK.ts

```diff
@@ -22,17 +24,20 @@ const ENGINE_RUNTIMES: { [key: string]: string } = {
   10: "nodejs10",
 };
 
-export const ENGINES_FIELD_REQUIRED_MSG = clc.bold(
-  "Engines field is required in package.json but none was found."
-);
 export const UNSUPPORTED_NODE_VERSION_MSG = clc.bold(
   `package.json in functions directory has an engines field which is unsupported. ` +
     `The only valid choices are: ${clc.bold('{"node": "8"}')} and ${clc.bold('{"node": "10"}')}. ` +
     `Note that Node.js 6 is now deprecated.`
 );
-export const DEPRECATION_WARNING_MSG =
+
+export const NODE6_DEPRECATION_WARNING_MSG =
   clc.bold.yellow("functions: ") +
-  "Deploying functions to Node 6 runtime, which is deprecated. Node 8 is available " +
+  "Deploying functions to Node 6 runtime, which is deprecated. Node 10 is available " +
```

Comment:
Optional - https://cloud.google.com/functions/docs/concepts/nodejs-6-runtime says 4-22-2020 is when GCF will fully stop supporting Node 6. It might be more appropriate top do all the cleanup in one separate PR, but it would be nice to add that date to this warning message.

Reply:
I'm removing all Node version messaging changes from this PR, in lieu of Michael's PR.